lib/driver.js

```diff
@@ -287,6 +293,12 @@ class AndroidDriver extends BaseDriver {
     await this.bootstrap.sendAction("compressedLayoutHierarchy", {compressLayout: compress});
   }
 
+  // Set UiAutomator Configurator related configurations
+  // value sould be like {"method":"setWaitForIdleTimeout","value":5000}
+  async setConfiguratorConfig (value) {
+    await this.bootstrap.sendAction("configurator", JSON.parse(value));
```

Comment:
ah yes I see you're expecting JSON objects. I'd love to see a bit of nicer error handling, something like:

```js
let param;
try {
  param = JSON.parse(value);
} catch (e) {
  log.errorAndThrow(`Configurator values must be valid JSON! Original error: ${e}`);
}
await this.bootstrap.sendAction("configurator", param);
```